README.md

```diff
@@ -1,14 +1,397 @@
+# Microsoft Identity Web
 
-# Contributing
+[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/ASP.NET%20Core%20Web%20App%20tutorial)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=819)
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+This library contains a set of reusable classes useful in ASP.NET Core:
 
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+- [Web applications](#web-apps) that sign in users, and, optionally, call web APIs
+- [Protected web APIs](#web-apis), which, optionally call downstream web APIs

```

Comment:
Broken links for me.

Reply:
I just checked #web-apis should work - when in the branch?